### PR TITLE
Change Username Font

### LINKED ISSUE #303: Use sans-serif font for username at the top right corner of the page
## Description

Some usernames will be hard to read with the decorative font.

### LINKED ISSUE #458: Missing period in English welcome text
## Description

There is a missing period in the English welcome text in the `translation_en.json` file.

## Expected Behavior

There should be a period at the end of the sentence.

## Actual Behavior

There isn't a period at the end of the sentence.

## Steps to Reproduce

1. Open home page.

## Possible Fix

Add a period at the end of the string in the `translation_en.json` file.

# Other

### PR DESCRIPTION
**Pull request by pascwhale**
Closes #303 
Closes #458 

## Description
- Change Login and Username font to Roboto on welcome page
- Add missing period at the end of the English translation of the welcome text

### CODE REVIEW

File: resources/js/Layouts/ApplicationLayout/ApplicationLayout.tsx [hunk 1]
```diff
@@ -71,7 +71,9 @@ const Username = styled(Typography)`
         margin-top: 10px;
     }
     margin: 10px 6px 0px 0px;
-    font-size: 14px;
+    // font-size: 14px;
```
Comment (willyyhuang) [inline resources/js/Layouts/ApplicationLayout/ApplicationLayout.tsx:74]: can you remove this commented line?

**Review by willyyhuang — COMMENTED** [review]
other than the commented line, everything looks good

File: resources/js/Layouts/ApplicationLayout/ApplicationLayout.tsx (hunk 1 above)
Comment (pascwhale) [reply resources/js/Layouts/ApplicationLayout/ApplicationLayout.tsx:74]: Oh, I forgot that I did that. Thanks

**Review by willyyhuang — APPROVED** [review]

**Comment by pascwhale** [comment]
Removed commented line and updated tests to respect new case of some words.

**Pull request merged**